src/store/messages/saga.ts

```diff
@@ -111,6 +112,10 @@ export function* fetch(action) {
       hasLoadedMessages: true,
     })
   );
+
+  // Publish a system message across the channel
+  const channel = yield call(conversationsChannel);
+  yield put(channel, { channelId });
```

Comment:
Let's add a type here. I know we still have a few of these events that don't specify a type but we should try to make the new ones use it. It allows us to filter properly in the listeners (rather than listening for '*')

Reply:
makes sense, updated. Will keep in mind for the new ones.